maintainers/maintainer-list.nix

```diff
@@ -12492,4 +12492,14 @@
     github = "rski";
     githubId = 2960312;
   };
+  zenithal = {
```

Comment:
Please put this into a separate commit with the message ``maintainers: add NAME`` and sort it alphabetically.

Reply:
Done.

It seems that `maintainers/maintainer-list.nix` is not sorted, so I inserted myself in the proper place and left others alone.